.github/workflows/ci.yml

```diff
@@ -14,6 +14,8 @@ jobs:
           - 2.5
           - 2.6
           - 2.7
+          - '3.0'
```

Comment:
Why is this one quoted?

Reply:
3.0 without quoting is interpreted as the number 3.
In that case, ruby/setup-ruby will use 3.1 not 3.0.

Please see https://github.com/ruby/setup-ruby/issues/252 for more details.